src/styles/blocks/header.scss

```diff
@@ -0,0 +1,15 @@
+$headerHight: 60px;
```

Comment:
There is a typo in the variable name `$headerHight`. It should be `$headerHeight` to correctly represent the word 'height'.